src/apis/KapacitorApi.ts

```diff
@@ -24,7 +25,19 @@ export default class KapacitorApi {
         return responseJSON.tasks;
     }
 
-    public static async updateTaskVariables(taskName: string, taskVariables: any): Promise<string> {
+    public static async templates(): Promise<KapacitorTask[]> {
+        const response = await fetch(`http://82.140.0.78:9092/kapacitor/v1/templates`);
```

Comment:
Please move the kapacitor URL to the config.ts as well.

Reply:
Done